README.md

```diff
@@ -19,3 +19,60 @@ Click on image to try the timer yourself
 - Works in the background (you don't need to keep the window in front)
 - No installation needed, runs [just here](https://jschirrmacher.github.io/timer)
 - No fees, no signup, no tracking
+- optimized for OBS
+
+## How to use with [OBS](https://obsproject.com/)
+
+With OBS, you can use this timer in your next live stream or video conference. To do so, add the timer as `Browser` Element.
+The standard user defined CSS added by OBS will already render the background transparent.
+To remove the text elements from the timer, style this elements with `display: none`:
+
+```
+body { 
+    background-color: rgba(0, 0, 0, 0); 
+    margin: 0px auto; 
+    overflow: hidden; 
+}
+#info, #goGitHub, #currentTime {
+    display: none;
+}
+```
+
+Feel free to change additional styles to fit your design.
+
+When use use the timer installed on a web server, you can just use the URL and append the parameter `duration` with the duration in minutes.
```

Comment:
This might be a typo. Should it be `When you use...`?

Reply:
I guess the typos are resolved now.